wesnoth-wof.po

```diff
@@ -2374,7 +2400,7 @@ msgstr "Vpřed!"
 #. [message]: speaker=Poseira
 #: data/campaigns/Winds_of_Fate/scenarios/04_Journey.cfg:355
 msgid "Nemas, you are violating a direct order! Return to your keep at once!"
-msgstr "Porušujete přímý rozkaz, Nemasi! Okamžitě se vraťte do své pevnosti!"
+msgstr "Porušujete přímý rozkaz, Nemasi! Okamžitě se vrať do své pevnosti!"
```

Comment:
Je to záměr to množné číslo v Porušujete, nebo má být Porušuješ?

Reply:
Aha. Mám nějaký mišmaš v tom, jestli si vykají nebo tykají. Původně jsem to překládal spíš na vykání, pak mi ale přišlo, že vztah mezi nimi je bližší (Poseira očividně chce Nemase chránit) a předělával jsem to na tykání. A samozřejmě jsem to udělal nedůsledně. Můžeš to klidně unifikovat sám na to, co ti přijde logičtější.